packages/web_benchmarks/lib/client.dart

```diff
@@ -50,8 +48,8 @@ Future<void> runBenchmarks(Map<String, RecorderFactory> benchmarks) async {
   html.window.location.reload();
 }
 
-Future<void> _runBenchmark(String benchmarkName) async {
-  final RecorderFactory recorderFactory = _benchmarks[benchmarkName];
+Future<void> _runBenchmark(String? benchmarkName) async {
```

Comment:
If the function unconditionally unwraps an argument—which the first line does here—then it's not actually optional.